PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+# [Grove - Ultrasonic Distance Sensor](http://wiki.seeedstudio.com/Grove-Ultrasonic_Ranger/) on A0
+# [Grove - 16x2 LCD](http://wiki.seeedstudio.com/Grove-16x2_LCD_Series/) on I2C1
+# [Grove - Speaker Plus](http://wiki.seeedstudio.com/Grove-Speaker/) on UART2
+import time
+import subprocess
+import os
+import sys
+import wave
+import pyaudio
+from tqdm import tqdm
+try:
+    import iio
+except:
+    # By default the iio python bindings are not in path
+    sys.path.append('/usr/lib/python2.7/site-packages/')
+    import iio
+_SCALE_DEFS = [
+   'do.wav',
+   're.wav',
+   'me.wav',
+   'fa.wav',
+   'so.wav',
+   'la.wav',
+   'ti.wav'
+   ]
+def Play_Music(file):
+    # define stream chunk
+    chunk = 1024
+    # open a wav format music
+    f = wave.open(file,"rb")
```

Comment:
Can we move these files to /tmp ?

Reply:
if we move in /tmp , the file will disappear when we restart it.